lib/Xud.ts

```diff
@@ -50,15 +51,24 @@ class Xud {
       this.db = new DB(this.config.db);
       await this.db.init();
 
-      this.lndClient = new LndClient(this.config.lnd);
-      await this.lndClient.connect();
+      // setup LND clients and connect if configured
+      this.lndbtcClient = new LndClient(this.config.lndbtc);
+      if (!this.lndbtcClient.isDisabled()) {
```

Comment:
We don't need these `isDisabled` checks here because we already check for disabled status within the connect method, which is the better place for this sort of check I think.

Reply:
We need it here in order to avoid an error message that will be issued in case you try to connect().

Reply:
Ah yeah that's right, we can leave as is.

Reply:
Ah yeah you are right, we can leave this as is.